apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/close-button-override/index.js

```diff
@@ -4,28 +4,42 @@
  * External dependencies
  */
 import domReady from '@wordpress/dom-ready';
+import { __ } from '@wordpress/i18n';
+import './style.scss';
 
 domReady( () => {
 	const { closeButtonLabel, closeButtonUrl, editorPostType } = fullSiteEditing;
 
 	const editPostHeaderInception = setInterval( () => {
-		const closeButton = document.querySelector( '.edit-post-fullscreen-mode-close__toolbar a' );
+		// cycle through interval until header toolbar is found
+		const toolbar = document.querySelector( '.edit-post-header__toolbar' );
 
-		if ( ! closeButton ) {
+		if ( ! toolbar ) {
 			return;
 		}
 		clearInterval( editPostHeaderInception );
 
+		// add components toolbar with override class name (original will be hidden in ./style.scss)
+		const componentsToolbar = document.createElement( 'div' );
```

Comment:
We want to make sure that we only add this element for the page and wp_template_part post types. Currently, it adds the element for the normal post editor as well:

<img width="1501" alt="Screen Shot 2019-10-31 at 12 10 23 PM" src="https://user-images.githubusercontent.com/6265975/67978738-ff395a80-fbd7-11e9-933f-96f12cf4d2c1.png">

Reply:
Great catch!  I have added a condition near the top of index.js so that this does change anything for the posts editor.

I do wonder, however, if it would be worth adding this button override functionality to the posts editor.  That is, there is currently no back button when viewing the post editor in windowed mode via wp-admin.  If we want the back button in that scenario for the page editor, we would probably want that as well for the posts editor (although this would reach beyond where the FSEditor is currently active).